packages/@ember/canary-features/index.ts

```diff
@@ -20,6 +20,7 @@ export const DEFAULT_FEATURES = {
   EMBER_CUSTOM_COMPONENT_ARG_PROXY: true,
   EMBER_GLIMMER_SET_COMPONENT_TEMPLATE: true,
   EMBER_ROUTING_MODEL_ARG: true,
+  EMBER_GLIMMER_IN_ELEMENT: null,
```

Comment:
I think it's _probably_ fine to enable by default in this PR too, since it's not a major refactor. The feature flag is mostly to give us a way to turn it off in case we do run into major issues during `beta`. Will confirm though

Reply:
Ah right. Now that you are mentioning this, I remember that I wanted to ask about this 😝

Reply:
Sorry, seems I was mistaken. Assuming travis passes I think this is good to go!